virttest/qemu_vm.py

```diff
@@ -1900,6 +1900,15 @@ def sort_key(dev):
                               pci_bus=pci_bus)
                 iov += 1
 
+        # Add vsock device, cid 0-2 are reserved by system
+        default_cid = 3
+        for vsock in params.objects('vsock'):
```

Comment:
Suggest the name `vsocks` if each vm can have more than one vsock.

Reply:
Sure, that's what I meant.